bowling/swift/Bowling/README.md

```diff
@@ -14,19 +16,26 @@ In the tenth frame a player who rolls a spare or strike is allowed to roll the e
 balls to complete the frame.  However no more than three balls can be rolled in
 tenth frame.
 
-Original Source: http://butunclebob.com/files/downloads/Bowling%20Game%20Kata.ppt
+Sources:
+- [kata logs](http://kata-log.rocks/bowling-game-kata)
+- [Power point](http://butunclebob.com/files/downloads/Bowling%20Game%20Kata.ppt)
```

Comment:
Nit: it's called `PowerPoint`, not `Power point` (source: [Microsoft](https://products.office.com/en-gb/powerpoint))